### DISCUSSION TITLE
How to customize Page Meta part?

### DISCUSSION THREAD

**Discussion by TetraTheta** [hbstack/.github#29] [Q&A] [answered]
I'm asking here because I couldn't find related repository for this problem.

---

Currently, page meta consist of three parts: 'Post Date', 'Reading Time', 'Post Section'.

I want to modify 'Post Section' part.  
Currently, 'Post Section' only shows top-level category. If a post is located as `/cat1/cat1-1/post1/index.md`, it will show link to `/cat1`.

I want to make this behavior changed. Similar to breadcrumb, it will show 'cat1 / cat1-1' instead. both `cat1` and `cat1-1` will have link to corresponding URL, `mysite.com/cat1/` and `mysite.com/cat1/cat1-1/`.

But I couldn't find which module processes this part.

How should I modify codes to achieve this behavior?

**Comment by razonyang** [comment, marked as answer]
You can use the GitHub search on the hbstack org with the classname or something unique, then you will find the code.
https://github.com/search?q=org%3Ahbstack+hb-blog-post-section&type=code

And then create a layout/template with same filepath, in this case, [layouts/partials/hb/modules/blog/post/meta/first-section.html](https://github.com/hbstack/blog/blob/8cdb7c8831e52f3f0aace3d7ba2eb7f071a31b50/layouts/partials/hb/modules/blog/post/meta/first-section.html#L3). Copy the content and modify it, according to the Hugo look up order, you can override any theme/module's templates and files.

**Reply by TetraTheta** [reply]
Sorry for my incompetance. I didn't thought about that I can search class name for that part. I'm awefully sorry.

**Reply by razonyang** [reply]
No problem.
